index.html

```diff
@@ -4089,7 +4089,7 @@ <h3><code>forms</code></h3>
 
   <p>
   When <code>forms</code> is present at the top level, it can be used to describe meta interactions offered by a <a>Thing</a>.
-  For example, the operation types "readallproperties" and "writeallproperties" are for meta
+  For example, the operation types <code>"readallproperties"</code> and <code>writeallproperties"</code> are for meta
```

Comment:
Q1: Do we want surrounding quotes " ?
Q2: IF we want quotes the second one misses it. see <code>writeallproperties"

Reply:
Ah it should be without the quotes, fixing